### PR TITLE
copytext / ui updates

### PR DESCRIPTION
**Pull request by macool**
includes points 5, 6, 7 & 8 from provided document

### CODE REVIEW

**Comment by macool** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;YGQ6smF1&#x2F;643-qae-support-successful-business-winners-press-book-entry-screens"> QAE Support: Successful Business Winners Press Book Entry Screens</a>

File: db/schema.rb [hunk 1]
```diff
@@ -312,7 +312,7 @@
     t.boolean  "approved",         default: false
     t.datetime "created_at",                       null: false
     t.datetime "updated_at",                       null: false
-    t.string   "first_name"
+    t.text     "first_name"
```
Comment (matthewford) [inline db/schema.rb]: Where is this from? Is this needed?

Comment (matthewford) [inline db/schema.rb]: A string will be fine, we can revert this change

Comment (matthewford) [inline db/schema.rb]: perhaps rename it to name
